README.md

```diff
@@ -151,12 +188,13 @@ public class MainApplication extends Application implements ReactApplication {
   }
 }   
 ```
-
-
 ## Development
 
 See [development.md](./docs/development.md) for development docs.
```

Comment:
Can we move this one under Contributing as well, e.g.:
Contributions are welcomed! See the [CONTRIBUTING](CONTRIBUTING.md) and [development](./docs/development.md) guides for more information.